packages/vue-app/src/index.js

```diff
@@ -8,15 +8,15 @@ export const template = {
     'App.js',
     'client.js',
     'index.js',
-    'middleware.js',
+    // 'middleware.js',
     'router.js',
     'router.scrollBehavior.js',
     'server.js',
     'utils.js',
     'empty.js',
-    'components/nuxt-build-indicator.vue',
+    // 'components/nuxt-build-indicator.vue',
```

Comment:
For long term: I wanted to export a function instead that takes builder instance to dynamically provide template. Refactoring logic of feature related options here.